view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,272 @@
+							{{/if}}
+
+							{{if $order_users != $th_users.3.1}}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.3.1}}">
+										&#8597; {{$th_users.3.0}}</a> : {{$u.login_date}}</p>
+							{{/if}}
+
+							{{if $order_users != $th_users.4.1}}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.4.1}}">
+										&#8597; {{$th_users.4.0}}</a> : {{$u.lastitem_date}}</p>
+							{{/if}}
+
+							{{if in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1]) }}
+								<p><a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th_users.5.1}}">
+										&#8597; {{$th_users.5.0}}</a> : {{$u.page_flags}} {{if $u.is_admin}}({{$siteadmin}}){{/if}} {{if $u.account_expired}}({{$accountexpired}}){{/if}}</p>
+							{{/if}}
+
+						</td>
+						<td class="text-right">
+							{{if $u.is_deletable}}
+								<a href="{{$baseurl}}/admin/users/block/{{$u.uid}}?t={{$form_security_token}}" title='{{if $u.blocked}}{{$unblock}}{{else}}{{$block}}{{/if}}'>
+									{{if $u.blocked==0}}
+									<i class="fa fa-ban" aria-hidden="true"></i>
+									{{else}}
+									<i class="fa fa-circle-o" aria-hidden="true"></i>
+									{{/if}}
+								</a>
+								<a href="{{$baseurl}}/admin/users/delete/{{$u.uid}}?t={{$form_security_token}}" title='{{$delete}}' onclick="return confirm_delete('{{$u.name}}')"><i class="fa fa-trash" aria-hidden="true"></i></a>
+							{{else}}
+								&nbsp;
+							{{/if}}
+						</td>
+					</tr>
+				{{/foreach}}
+				</tbody>
+			</table>
+			<div class="panel-footer text-right">
+						<button type="submit" name="page_users_block" class="btn btn-warning">	<i class="fa fa-ban" aria-hidden="true"></i> {{$block}} / <i class="fa fa-circle-o" aria-hidden="true"></i> {{$unblock}}</button>
+						<button type="submit" name="page_users_delete" class="btn btn-danger" onclick="return confirm_delete_multi()"><i class="fa fa-trash" aria-hidden="true"></i> {{$delete}}</button>
+			</div>
+		{{else}}
+			<div class="panel-body text-center bg-danger">NO USERS?!?</div>
+		{{/if}}
+		</div>
+
+
+
+	</form>
+
+
+
+
+
+<!--
+	**
+	*
+	*		DELETED Users table
+	*
+	**
+-->
+	{{if $deleted}}
+	<div class="panel panel-default">
+		<div class="panel-heading"><h3 class="panel-title">{{$h_deleted}}</h3></div>
+		<table id='deleted' class="table table-hover">
+			<thead>
+			<tr>
+				<th></th>
+				{{foreach $th_deleted as $k=>$th}}
+					{{if in_array($k,[0,1,5])}}
+					<th>{{$th}}</th>
+					{{/if}}
+				{{/foreach}}
+			</tr>
+			</thead>
+			<tbody>
+			{{foreach $deleted as $u}}
+				<tr>
+					<td><img src="{{$u.micro}}" title="{{$u.nickname}}"></td>
+					<td><a href="{{$u.url}}" title="{{$u.nickname}}" >{{$u.name}}</a></td>
+					<td>{{$u.email}}</td>
+					<td>{{$u.deleted}}</td>
+				</tr>
+			{{/foreach}}
+			</tbody>
+		</table>
+	</div>
+{{/if}}
+
+
+
+<!--
```

Comment:
Smarty comments are done with `{{*` and `*}}`
e.g.
`{{* NEW USER Form *}}`

Reply:
but.. those are html comment :-)

Reply:
How useful are they to anyone looking at the page source?

Reply:
The page is composed by two / tree very similar tables. I've added the comments to find them easily with firefox dev tools...

Reply:
Ha, I see.